ci-operator/step-registry/upi/conf/vsphere/ova/windows/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
```

Comment:
We need someone from the splat team here.

Reply:
cc: @rvanderp3   @jcpowermac

Reply:
Addressed.